synapse/storage/databases/main/event_federation.py

```diff
@@ -726,17 +726,39 @@ def _get_auth_chain_difference_txn(
     async def get_backfill_points_in_room(
         self,
         room_id: str,
+        current_depth: int,
+        limit: int,
     ) -> List[Tuple[str, int]]:
         """
         Gets the oldest events(backwards extremities) in the room along with the
-        approximate depth. Sorted by depth, highest to lowest (descending).
+        approximate depth. Sorted by depth, highest to lowest (descending) so the closest
+        events to the `current_depth` are first in the list.
+
+        We use this function so that we can compare and see if a client's
+        `current_depth` at their current scrollback is within pagination range
+        of the event extremities. If the `current_depth` is close to the depth
```

Comment:
I'm afraid I don't really understand this sentence. 

"We use this function so that ...": as opposed to some other function?

"we can compare": compare what?

what are "event extremities"?

Generally I think this text probably makes more sense at the call site.